requirements.txt

```diff
@@ -1,4 +1,5 @@
 numpy >=1.15
 pandas
 boto3
-s3fs
\ No newline at end of file
+s3fs
+pylint
```

Comment:
I think it'd be better to use `black` and run a lint test as part of the CI. Pylint throws errors for a combination of style errors